vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_authorization.cc

```diff
@@ -14,6 +14,7 @@
 #include "bat/ledger/internal/ledger_impl.h"
 #include "bat/ledger/internal/logging/event_log_keys.h"
 #include "bat/ledger/internal/uphold/uphold_util.h"
+#include "bat/ledger/option_keys.h"
```

Comment:
This include may not be needed after BAP was removed

Reply:
Resolved in 829eadc955d034cd2c4bf783bfc0e9455a73ea1a.